subdomains/docs/_guide/getting-started.md

```diff
@@ -16,6 +16,12 @@ For [bash](https://www.gnu.org/software/bash/), [zsh](https://www.zsh.org/), and
 - Set the `VOLTA_HOME` variable to `$HOME/.volta`
 - Add `$VOLTA_HOME/bin` to the beginning of your `PATH` variable
 
+You need set your default version of node before running any of Volta's commands, to use the LTS version of node run:
+
+```bash
+volta install node
+```
+
```

Comment:
I really love having a call to action for selecting your default version, as that is very often going to be an important step. I actually think this would be good to be in it's own section _after_ installation. It might take some reworking of the headings - Maybe we have a section titled "Install Volta" with subsections for Unix/Windows, and then another section titled "Select a default Node" that contains this part.

As I mentioned in the issue, it's not entirely true that you need to have a default version before running Volta's commands—In a pinned project it should work just fine even without a default version. Could we update the language to be something like:

> Select a default version of Node—this is the version that Volta will use everywhere outside of projects that have a pinned version.

It would also be good to include the full command (with a specific version) as well, maybe something like:

> To select a specific version of Node, run... Or to use the latest LTS version, run...

Reply:
Pushed. Let me know how this looks! Was there a reason for "Getting started" and "Understanding Volta" being separate like they are? To me it seems more like the current "Getting Started" is more commonly known as just "Installation" and "Understanding Volta" is the "Getting Started" section. In the sense that the latter delves into the core usage of the tool :) Thank you!